convert-hf-to-gguf.py

```diff
@@ -3128,11 +3129,12 @@ def main() -> None:
         if args.vocab_only:
             logger.info("Exporting model vocab...")
             model_instance.write_vocab()
-            logger.info("Model vocab successfully exported.")
+            logger.info(f"Model vocab successfully exported to {model_instance.fname_out}")
         else:
             logger.info("Exporting model...")
             model_instance.write()
-            logger.info("Model successfully exported.")
+            out_path = os.path.dirname(model_instance.fname_out) + '/' if is_split else model_instance.fname_out
```

Comment:
`model_instance.fname_out` is a `Path` (from `pathlib`), so instead of `os.path.dirname`, there's also the `parent` property:
```suggestion
            out_path = f"{model_instance.fname_out.parent}/" if is_split else model_instance.fname_out
```

Reply:
Updated now :+1:

Reply:
I think the trailing slash should be `{os.sep}`. I don't have a Windows development environment, so I'm not sure about this, but from jumping around its source, `pathlib` uses the separator of the host OS when printing with `str(some_path)` (like implicitly in `f"{some_path}"`), and so always using a slash (`/`) might look weird on that OS.

If that's not true, feel free to disregard this.

Reply:
Good point! I'll take a look what this looks like on Windows.

Reply:
This does not look correct on Windows (just using `/`) and I'll update it to use `os.sep` shortly. Thanks!